### ISSUE TITLE
Link animation direction 

### ISSUE THREAD

**Issue by ahsanbagwan** [projectstorm/react-diagrams#286]
I have links and nodes for which the animation direction seems to be default from left to right. I want the direction to be the other way also. This animation is out of the box on hover and selected actions.

Anyway to change the direction of animation in links?

**Comment by kitoki** [comment]
yeah, for me, it only when click at least once, if not, even when hover, still left to right.

**Comment by wamujlb** [comment]
@ahsanbagwan I have used this approach: 

```
	/**
     * Get direction of the link
     * Useful for animation
     * @method getDirection
     * @param source - source port
     * @param target - target port
     * @return true if direction should be reversed
     */
    getDirection(source: Object, target: Object): boolean {
        const difX = source.x - target.x,
            difY = source.y - target.y,
            isHorisontal = Math.abs(difX) > Math.abs(difY);

        return isHorisontal ? difX > 0 : difY > 0;
    }
```

**Comment by flieks** [comment]
Do you mean this link animation ?

![image](https://user-images.githubusercontent.com/5032389/86533502-f8cf5100-bed1-11ea-962c-4af85c62f378.png)

For me also the lines are moving from IN to OUT ports. But i want them to move from OUT to IN. How can we accomplish this ?

**Comment by flieks** [comment]
Nevermind, my mistake i had to change
portOut.link(portIn);
to 
portIn.link(portOut);
